src/lib/y2network/proposal_settings.rb

```diff
@@ -140,6 +147,10 @@ def create_instance
 
   private
 
+    def package_selected?(name)
+      Y2Packager::Resolvable.any?(kind: :package, name: name, status: :selected)
```

Comment:
What about the upgrade case? What if the package is already installed? Or is this proposal is only for new installations?

Reply:
During upgrade the network is not touched at all, this is only for installation & autoinstallation.